doc/api/v0.yaml

```diff
@@ -175,3 +198,22 @@ components:
         fund_id:
           type: integer
           format: int32
+
+    Challenge:
+      properties:
+        title:
+          type: string
+        description:
+          type: string
+        extended_description:
+          type: string
```

Comment:
Does this come from the IdeaScale data model?

Reply:
Mostly based on the UI mock-ups I was provided with. Need to clarify how do they agree with the IdeaScale.